app/src/main/java/org/oppia/android/app/administratorcontrols/AdministratorControlsViewModel.kt

```diff
@@ -68,25 +68,22 @@ class AdministratorControlsViewModel @Inject constructor(
     deviceSettings: DeviceSettings
   ): List<AdministratorControlsItemViewModel> {
 
-    val itemViewModelList: MutableList<AdministratorControlsItemViewModel>
+    val itemViewModelList: MutableList<AdministratorControlsItemViewModel> =
+      emptyList<AdministratorControlsItemViewModel>().toMutableList()
```

Comment:
You can instead do: ``mutableListOf<AdministratorControlsItemViewModel>()``

You also don't need to specify the val's type in this context since it can be inferred.